<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopoDialogBase</class>
 <widget class="QDialog" name="TopoDialogBase">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>681</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Topo</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="3" colspan="3">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QDoubleSpinBox" name="transversal">
     <property name="maximum">
      <double>999.990000000000009</double>
     </property>
     <property name="value">
      <double>30.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QDoubleSpinBox" name="montanhosoMax">
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QDoubleSpinBox" name="planoMax">
     <property name="value">
      <double>8.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="6">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>[Nota Legal] As marcas Google® Maps e Bing® Maps são marcas comerciais pertencentes a Google Inc. e Microsoft Inc. respectivamente,
 e são das mesmas os direitos do recurso de abertura de mapas. Tambem agradecemos ao projeto OpenLayers.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_12">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Espaçamento transversal de cada lado (m)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Sistema de coordenada do projeto:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="txtCRS"/>
   </item>
   <item row="4" column="5">
    <widget class="QDoubleSpinBox" name="onduladoMax">
     <property name="value">
      <double>20.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Taxa i para Ondulado(%)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Separador CSV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtCSV">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>;</string>
       </property>
       <property name="maxLength">
        <number>1</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="3">
    <widget class="QDoubleSpinBox" name="onduladoMin">
     <property name="value">
      <double>8.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QDoubleSpinBox" name="estacas">
     <property name="maximum">
      <double>999.990000000000009</double>
     </property>
     <property name="value">
      <double>20.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Taxa i para Montanhoso(%)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Padrão Medidas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboUnits">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Metros</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kilometros</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Milimetros</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>até</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>até</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_11">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Distanciamento entre estacas (m)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="9">
    <widget class="QTableWidget" name="tableCRS">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Taxa i para Plano(%)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>até</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="5">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Servidor de mapas</string>
     </property>
     <widget class="QComboBox" name="comboMap">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>451</width>
        <height>27</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Selecione um servidor de mapas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Google® Maps Satelite</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Google® Maps Street</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bing® Maps Satelite</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bing® Maps Street</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QDoubleSpinBox" name="planoMin"/>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Configurações do projeto:</string>
     </property>
     <widget class="QComboBox" name="comboClasse">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>421</width>
        <height>27</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Selecione a classe do projeto</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0 - Via Expressa, controle total de acesso</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1(A) - Pista dupla, controle parcial de acesso</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1 (B) - Pista simples, controle parcial de acesso</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2 - Pista simples</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3 - Pista simples</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4 (A) - Pista simples</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4 (B) - Pista simples</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QDoubleSpinBox" name="montanhosoMin">
     <property name="value">
      <double>20.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionOpenFolderAction">
   <property name="text">
    <string>openFolderAction</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>TopoDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>814</x>
     <y>586</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>TopoDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>814</x>
     <y>586</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtCRS</sender>
   <signal>textChanged(QString)</signal>
   <receiver>tableCRS</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
